### PR TITLE
Update and rename toggl from 7.5.254 to toggl-track 7.5.260

### PR DESCRIPTION
**Pull request by ran-dall**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Created with [`cask-repair`](https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask).

### CODE REVIEW

**Comment by core-code** [comment]
this has been renamed to "Toggl Track.app"

@vitorgalvao  i am not sure if we need to rename the cask too ... it wasn't even named "toggl-desktop" before?

**Comment by miccal** [comment]
> this has been renamed to "Toggl Track.app"
> 
> @vitorgalvao i am not sure if we need to rename the cask too ... it wasn't even named "toggl-desktop" before?

This should be re-named to `toggl-track` to follow the [token reference.](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md)

File: Casks/toggl.rb [hunk 1]
```diff
@@ -5,8 +5,9 @@
   # github.com/toggl-open-source/toggldesktop/ was verified as official when first introduced to the cask
   url "https://github.com/toggl-open-source/toggldesktop/releases/download/v#{version}/TogglDesktop-#{version.dots_to_underscores}.dmg"
   appcast "https://toggl-open-source.github.io/toggldesktop/assets/releases/darwin_stable_appcast.xml"
-  name "TogglDesktop"
+  name "Toggl Desktop"
+  name "Toggl Track"
   homepage "https://www.toggl.com/toggl-desktop/"
```
Comment (miccal) [inline Casks/toggl.rb:10]:
```
homepage "https://www.toggl.com/track/toggl-desktop/"
```

**Review by core-code — DISMISSED** [review]

File: Casks/toggl-track.rb [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+cask "toggl-track" do
+  version "7.5.260"
+  sha256 "4c001d7d9940ccd5563be1f03773ceeaca46efaf07be30819e04746e42bbeee3"
 
   # github.com/toggl-open-source/toggldesktop/ was verified as official when first introduced to the cask
   url "https://github.com/toggl-open-source/toggldesktop/releases/download/v#{version}/TogglDesktop-#{version.dots_to_underscores}.dmg"
   appcast "https://toggl-open-source.github.io/toggldesktop/assets/releases/darwin_stable_appcast.xml"
-  name "TogglDesktop"
-  homepage "https://www.toggl.com/toggl-desktop/"
+  name "Toggl Track"
```
Comment (vitorgalvao) [inline Casks/toggl-track.rb:8]: `name` is used for search. `Desktop` isn’t a useful term, so not necessary to keep both names.

**Comment by vitorgalvao** [comment]
> @vitorgalvao i am not sure if we need to rename the cask too ... it wasn't even named "toggl-desktop" before?

`desktop` is purposefully removed for being a common and unhelpful word (every cask is “desktop”).

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**